tests/TestCli.cpp

```diff
@@ -511,35 +505,31 @@ void TestCli::testClip()
 
     future.waitForFinished();
 
-    setInput("a");
-    execCmd(clipCmd, {"clip", m_dbFile->fileName(), "--totp", "/Sample Entry", "0"});
-    QVERIFY(m_stderr->readAll().contains("Invalid timeout value 0.\n"));
-
     setInput("a");
     execCmd(clipCmd, {"clip", m_dbFile->fileName(), "--totp", "/Sample Entry", "bleuh"});
     QVERIFY(m_stderr->readAll().contains("Invalid timeout value bleuh.\n"));
 
     setInput("a");
-    execCmd(clipCmd, {"clip", m_dbFile2->fileName(), "--totp", "/Sample Entry"});
```

Comment:
@droidmonkey do we already have a unit test for the `clip` command and unicode in the password (characters with multiple graphemes)?

Reply:
Need to add that

Reply:
@louib troubling, but not surprising, on Windows clip.exe cannot take UTF-8 input....

Reply:
Maybe we could test it only on Linux and Mac